nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -768,6 +798,7 @@ define([
             busEventManager.add(runStatusListener);
             cell.execute();
             updateState('launching');
+            switchToTab('viewConfigure');
```

Comment:
The configure tab stays active, even though according to the `visibleTabs` and `enabledTabs` in the bulk app states object say that configure should NOT be active. This is a hacky fix.